share/spice/syntax/syntax.js

```diff
@@ -0,0 +1,36 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_syntax = function(api_result) {
+        // Validate the response (customize for your Spice)
+        if (!api_result || api_result.error || api_result.length == 0) {
+            return Spice.failed('syntax');
+        }
+        var query = DDG.get_query().replace("syntax", '');
+        // Render the response
+        Spice.add({
+            id: "syntax",
+            // Customize these properties
+            name: "Programming",
+            data: api_result[0],
+            meta: {
+                sourceName: "SyntaxDB.com",
+                sourceUrl: 'https://syntaxdb.com/reference/search?search=' +  query,
```

Comment:
The reason I use the dynamically generated `sourceUrl` is because I want them to get the respective results page for their query when they click `More at SyntaxDB`.